src/main/java/jp/co/future/uroborosql/context/SqlContextFactoryImpl.java

```diff
@@ -282,19 +306,21 @@ public SqlContextFactory setEnumConstantPackageNames(final List<String> enumCons
 	 *
 	 * @see SqlContextFactory#setSqlFilterManager(SqlFilterManager)
 	 */
+	@Deprecated
 	@Override
 	public SqlFilterManager getSqlFilterManager() {
-		return sqlFilterManager;
+		return getSqlConfig().getSqlFilterManager();
 	}
 
 	/**
 	 * {inheritDoc}
 	 *
 	 * @see SqlContextFactory#setSqlFilterManager(SqlFilterManager)
 	 */
+	@Deprecated
 	@Override
 	public void setSqlFilterManager(final SqlFilterManager sqlFilterManager) {
-		this.sqlFilterManager = sqlFilterManager;
+		// do nothing
```

Comment:
@HidekiSugimoto189 
Here, it is kinder to output a warning log.

Reply:
@shout-star 
I added warning log.